modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/util/JSONUtil.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.translator.deepl.internal.util;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.net.URISyntaxException;
+
+/**
+ * JSON Utility
+ *
+ * @author Yasuyuki Takeo
+ */
+public class JSONUtil {
+
+	/**
+	 * Map json to the actual object
+	 */
```

Comment:
Either write proper javadoc (look for examples in existing code), or remove the comments. If you decide to write it, add a `@review` pseudo tag so that it can be reviewed.